tests/test_blobstore.py

```diff
@@ -108,13 +112,14 @@ def test_gs_md5(self):
     def test_put_get_delete(self):
         key = f"{uuid4()}"
         expected_data = os.urandom(1021)
-        for bs in (local_blobstore, gs_blobstore):
+        for bs in (local_blobstore, gs_blobstore, azure_blobstore):
             with self.subTest(blobstore=bs, key=key):
-                bs.blob(key).put(expected_data)
-                self.assertEqual(expected_data, bs.blob(key).get())
-                bs.blob(key).delete()
+                client = bs.blob(key)
```

Comment:
This seems like a blob instead of a client?

Reply:
it's the client object for doing stuff with the blob